### ISSUE TITLE
M1 chip incompatibility issues

### ISSUE THREAD

**Issue by jahbolero** [lovell/sharp#3105]
### What are you trying to achieve?
Image composition using Macbook M1. Reopening another ticket as updating to the latest sharp version didn't resolve the issue #3091 and #2286 was through a different use case.

Windows Test Case
OS:Windows 10
Processor:AMD Ryzen 7 4000
Node Version:14.16.1
Total Runtime:9 Minutes

Macbook M1 Air
OS:Monterey
Processor:Apple M1 
Node Version:15.6.0
Total Runtime:1 Hour, 15 Minutes

### Please provide a minimal, standalone code sample, without other dependencies, that demonstrates this question
```
const sharp = require('sharp');

const compositeImage = async (images, id) => {
  let inputArray = [];
  for (let i = 1; i < images.length; i++) {
    inputArray.push({ input: `${images[i]}` });
  }
  await sharp(`${images[0]}`).composite(inputArray).toFile(`output/${id}.png`);
};

(async () => {
  let imageArray = ['1.png', '2.png', '3.png', '4.png'];
  console.log('start:' + new Date().toLocaleTimeString());
  var i = 0;
  while (i < 1000) {
    await compositeImage(imageArray, i);
    i++;
  }
  console.log('end:' + new Date().toLocaleTimeString());
})();```

**Comment by lovell** [comment]
Node.js 15.6.0 reached EOL in June last year and is unsupported. As far as I know, it was made available as x64 only so will use Rosetta emulation on an M1.

As I said in #3091, please see the not yet released #2286

**Comment by jahbolero** [comment]
Works perfectly if using Node 17.6.0(Native to m1)

Thanks!
